src/data/all-abilities.ts

```diff
@@ -906,7 +906,7 @@ export function initAbilities() {
     new Ability(Abilities.PROTEAN, 6).attr(PokemonTypeChangeAbAttr),
     //.condition((p) => !p.summonData?.abilitiesApplied.includes(Abilities.PROTEAN)), //Gen 9 Implementation
     new Ability(Abilities.FUR_COAT, 6)
-      .attr(ReceivedMoveDamageMultiplierAbAttr, (_target, _user, move) => move.category === MoveCategory.PHYSICAL, 0.5)
+      .attr(StatMultiplierAbAttr, Stat.DEF, 2, (_user, target, _move) => !!target)
```

Comment:
NIT: No need for `_move` when it's **after** the required param
```suggestion
      .attr(StatMultiplierAbAttr, Stat.DEF, 2, (_user, target) => !!target)
```